package.nls.json

```diff
@@ -93,5 +93,33 @@
 "mssql.enableQueryHistoryFeature":"Should Query History feature be enabled",
 "mssql.intelliSense.lowerCaseSuggestions":"Should IntelliSense suggestions be lowercase",
 "mssql.persistQueryResultTabs":"Should query result selections and scroll positions be saved when switching tabs (may impact performance)",
-"mssql.queryHistoryLimit":"Number of query history entries to show in the Query History view"
+"mssql.queryHistoryLimit":"Number of query history entries to show in the Query History view",
+"mssql.query.maxXmlCharsToStore":"Number of XML characters to store after running a query",
+"mssql.tracingLevel":"[Optional] Log level for backend services. Azure Data Studio generates a file name every time it starts and if the file already exists the logs entries are appended to that file. For cleanup of old log files see logRetentionMinutes and logFilesRemovalLimit settings. The default tracingLevel does not log much. Changing verbosity could lead to extensive logging and disk space requirements for the logs. Error includes Critical, Warning includes Error, Information includes Warning and Verbose includes Information",
+"mssql.logRetentionMinutes":"Number of minutes to retain log files for backend services. Default is 1 week.",
+"mssql.logFilesRemovalLimit":"Maximum number of old files to remove upon startup that have expired mssql.logRetentionMinutes. Files that do not get cleaned up due to this limitation get cleaned up next time Azure Data Studio starts up.",

```

Comment:
I think there might be an extra `mssql.logRetentionMinutes` here

Reply:
I don't see two `mssql.logRetentionMinutes` settings in this file.  Could you refer me to the lines with the duplicate entries?

Reply:
In this line 100, we have
`"Maximum number of old files to remove upon startup that have expired mssql.logRetentionMinutes. Files that do not get cleaned up due to this limitation get cleaned up next time Azure Data Studio starts up."` 
Was the ` mssql.logRetentionMinutes` in the first sentence intended?

Reply:
It was copy/paste from ADS source code https://github.com/microsoft/azuredatastudio/blob/main/extensions/mssql/package.nls.json#L55, which also has the reference to `mssql.logRetentionMinutes`.  I think this is saying that setting controls the time the files will expire.  Not sure why this setting is even needed.  I'm just exposing the same settings we have in ADS.  I'll investigate separately if this setting is actually useful.